Source/Configuration/DefinitionOptionConfiguration/Matcher/TyphoonOptionMatcher.m

```diff
@@ -24,11 +24,13 @@ @interface TyphoonOptionMatch : NSObject
 @property (nonatomic, strong) id value;
 @property (nonatomic) Class memberClass;
 @property (nonatomic) Class kindClass;
+@property (nonatomic) Protocol *proto;
```

Comment:
Let's use the full name here - `protocol` (or, if it's reserved, something like `conformProtocol`.